Modules/_sqlite/module.c

```diff
@@ -346,6 +346,16 @@ static struct PyModuleDef _sqlite3module = {
         NULL
 };
 
+#define ADD_OBJECT(module, name, type)                             \
+do {                                                               \
+    Py_INCREF(&type);                                              \
+    if (PyModule_AddObject(module, name, (PyObject *)&type) < 0) { \
```

Comment:
You can use PyModule_AddType and you don't have to pass for type name for this case :)

Reply:
Nice, thanks! :) Will fix.